sources/packages/backend/libs/utilities/src/queue.constant.ts

```diff
@@ -23,8 +23,39 @@ export enum QueueNames {
   PartTimeMSFAAProcessResponseIntegration = "part-time-msfaa-process-response-integration",
   FullTimeMSFAAProcessResponseIntegration = "full-time-msfaa-process-response-integration",
   SFASIntegration = "sfas-integration",
+  ATBCResponseIntegration = "atbc-response-integration",
   ProcessNotifications = "process-notifications",
   ProcessArchiveApplications = "archive-applications",
   AssessmentWorkflowEnqueuer = "assessment-workflow-enqueuer",
   AssessmentWorkflowQueueRetry = "assessment-workflow-queue-retry",
 }
+
+/**
+ * List of scheduler queue names.
+ */
+export const schedulerQueueNames: QueueNames[] = [
```

Comment:
https://github.com/bcgov/SIMS/pull/3070/files#r1552559926

Reply:
Thanks for the hint. The non-schedulers records don't have `cron` expression in `queueConfiguration`, and we can use that to tell whether the queue is a scheduler or not.